README.md

```diff
@@ -1283,22 +1283,21 @@ typedef char IOCTL_BUF[4096];
 https://en.wikipedia.org/wiki/Page_cache
 
 * cache.files=off: Disables page caching. Underlying files cached,
-  mergerfs files are not.
+  mergerfs files are not. Implies `direct_io=true`, `kernel_cache=false`, and `auto_cache=false`.
```

Comment:
These are all long deprecated so I kinda don't want these mentioned really. At this point I should probably remove their mention entirely. It's been years since I deprecated them.